docs/csharp/language-reference/keywords/throw.md

```diff
@@ -27,7 +27,7 @@ The syntax of `throw` is:
 ```csharp
 throw [e]
 ```
-where `e` is an instance of a class derived from <xref:System.Exception?displayProperty=nameWithType>. The following example uses the `throw` statement to throw an @System.IndexOutOfRangeException if the argument passed to a method named `GetNumber` does not correspond to a valid index of an internal array.
+where `e` is an instance of a class derived from <xref:System.Exception?displayProperty=nameWithType>. The following example uses the `throw` statement to throw an <xref:System.IndexOutOfRangeException> if the argument passed to a method named `GetNumber` does not correspond to a valid index of an internal array.
```

Comment:
fullName instead of nameWithType? nameWithType for a type displays only the type name, not the fully qualified type name.